Tella/Scenes/Uwazi/ViewModel/UwaziViewModel.swift

```diff
@@ -0,0 +1,149 @@
+//
+//  UwaziViewModel.swift
+//  Tella
+//
+//  Created by Gustavo on 31/07/2023.
+//  Copyright © 2023 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+class UwaziTemplateViewModel: ObservableObject {
+    
+    var mainAppModel : MainAppModel
+
+    @Published var templates : [CollectedTemplate] = []
+    @Published var downloadedTemplates : [CollectedTemplate] = []
+    @Published var draftReports : [Report] = []
+    @Published var outboxedReports : [Report] = []
+    @Published var submittedReports : [Report] = []
+    @Published var selectedReport : Report?
+    @Published var selectedCell = Pages.templates
+    @Published var pageViewItems : [PageViewItem] = [
+        PageViewItem(title: LocalizableUwazi.uwaziPageViewTemplate.localized, page: .templates, number: ""),
+        PageViewItem(title: LocalizableReport.draftTitle.localized, page: .draft, number: "") ,
+        PageViewItem(title: LocalizableReport.outboxTitle.localized, page: .outbox, number: ""),
+        PageViewItem(title: LocalizableReport.submittedTitle.localized, page: .submitted, number: "")
+    ]
+    @Published var isLoading: Bool = false
+    @Published var serverURL : String
+    @Published var serverName : String
+    var subscribers = Set<AnyCancellable>()
+    var server: Server
+    var sheetItems : [ListActionSheetItem] { return [
+        ListActionSheetItem(imageName: "view-icon",
+                            content: self.selectedReport?.status?.sheetItemTitle ?? "",
+                            type: self.selectedReport?.status?.reportActionType ?? .viewSubmitted),
+        ListActionSheetItem(imageName: "delete-icon-white",
+                            content: LocalizableReport.viewModelDelete.localized,
+                            type: ReportActionType.delete)
+    ]}
+
+    var tellaData: TellaData? {
+        return self.mainAppModel.vaultManager.tellaData
+    }
+    
+    init(mainAppModel : MainAppModel, server: Server) {
+        
+        self.mainAppModel = mainAppModel
+        self.server = server
+        self.serverURL = server.url ?? ""
+        self.serverName = server.name ?? ""
+    }
+
+    func getTemplates() {
+        self.isLoading = true
+        Task {
+            guard let id = self.server.id else { return }
+            guard let locale = self.tellaData?.getUwaziLocale(serverId: id) else { return }
+            let template = try await UwaziServerRepository().handleTemplate(server: self.server, locale: locale)
```

Comment:
I think it's better to include the locale to the server see https://github.com/Horizontal-org/Tella-iOS/pull/74/files#r1334325206